### PR TITLE
fix(AutoSuggestBox): handle keyboard input for chosing suggestions. i…

### LINKED ISSUE #364: AutoSuggestBox.QuerySubmitted is not called on suggestion selection
## I'm submitting a...
- Bug report

## Current behavior
The `AutoSuggestBox.QuerySubmitted` event is not raised when an item of a suggested list is selected. `AutoSuggestBox.SuggestionChosen` is raised, though.

## Expected behavior
`AutoSuggestBox.QuerySubmitted` is raised when a suggestion is chosen.

## Environment
Nuget Package: 1.41.1

### PR DESCRIPTION
**Pull request by LHuss**
…nvoke proper events as per uwp.

fixes #364

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

#364

## What is the new behavior?

Keyboard input is handled to replicate uwp design. QuerySubmitted is raised on suggestion click, whereas on suggestion selection (via keyboard input), SuggestionChosen is raised.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs [hunk 1]
```diff
@@ -635,8 +635,6 @@ protected override void OnKeyDown(KeyRoutedEventArgs args)
 			{
 				case VirtualKey.Left:
 				case VirtualKey.Right:
-				case VirtualKey.Up:
-				case VirtualKey.Down:
```
Comment (davidjohnoliver) [inline src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs:639]: I wonder if these should be set as handled in the case of a TextBox that allows multiline input (ie `AcceptsReturn="True"`). Not necessarily a big deal in the context of the current PR, though.

Comment (LHuss) [reply src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs:639]: That's a good idea actually. I could theoretically just put the Up/Down cases first and break if acceptsreturn is false.

**Comment by LHuss** [comment]
> Did you try testing this? I can't recall if the UI Tests on WebAssembly support keyboard input.


I tested manually on wasm/android to see if anything broke but didn't check the unit tests. Will check them now.

**Comment by jeromelaban** [comment]
@davidjohnoliver there's no easy way to test keyboard input this way for now.

**Pull request merged**